mst/aiven-client.md/aiven-client-install.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Aiven Client for Managed Service for TimescaleDB
+excerpt: Install and configure Aiven Command Line tool for fully managed services on AWS, Azure, or GCP.
+product: mst
+---
+
+# Aiven Client for fully managed services
+
+Aiven Client is a command line tool for fully managed services. To use Aiven
+Client you need to first create an authentication token and then configure it to
+connect to your Managed Service for TimescaleDB using the command line.
+
+## Create an authentication token in Managed Service for TimescaleDB
+
+To connect to Managed Service for TimecaleDB using Aiven Client you need to
+create an authentication token.
+
+<procedure>
+
+### Creating an authentication token in Managed Service for TimescaleDB
+
+1.  Log in to your [Managed Service for TimescaleDB portal][mst-login].
+1.  Click `User Information` in the top right corner.
+1.  In the `User Profile` page, navigate to the `Authentication`tab.
+1.  Click `Generate Token`.
+1.  In the `Generate access token` dialog, type a descriptive name for the
+    token and leave the rest of the fields blank.
+1.  Copy the generated authentication token and save it.
+
+</procedure>
+
+## Install the Aiven Client
+
+Aiven Client is provided as a Python package, so you can install it on a Linux,
+MacOS, or Windows system using `pip`, if you have already installed Python. 
+
+Use this command: 
+
+```bash
+pip install aiven-client
+
```

Comment:
Need to add in the rest here ...

Reply:
Wasn't there another line after the code block?

Reply:
Not it is a just one line command

Reply:
Nope, found it:

```suggestion
For more information about installing the Aiven Client, see the
[Aiven][aiven-github] documentation.
```